translations/translationtool/src/translationtool.php

```diff
@@ -104,6 +125,34 @@ public function createPotFile() {
 		$this->deleteFakeFileForLocale();
 	}
 
+	private function checkMissingTranslations(string $entry, string $tmpfname) {
+		$translations = Gettext\Translations::fromPoFile($tmpfname);
+		$first=true;
+		foreach($translations as $translation) {
+			if (preg_match_all('/(^|[^a-zA-Z_]+)(t\([^\)]*\))/', $translation->getOriginal(), $matches)) {
+				$suspects = [];
+				foreach($matches[2] as $miss) {
+					if (preg_match('/["\']'.$this->name.'["\']/', $miss )) {
+						$suspects[] = $miss;
+					}
+				}
+				if (empty($suspects)) {
+					continue;
+				}
+				if ($first) {
+					echo '** Warning : Check potentially missing translations sentences : ' . $this->name . ' ' . $entry . PHP_EOL;
```

Comment:
You are french, right? :sunglasses: 
```suggestion
					echo '** Warning: Check potentially missing translations sentences: ' . $this->name . ' ' . $entry . PHP_EOL;
```

Reply:
Exactly!